src/polyfills/array-includes.js

```diff
@@ -44,6 +44,9 @@ function includes(value, opt_fromIndex) {
 */
 export function install(win) {
   if (!win.Array.prototype.includes) {
-    win.Array.prototype.includes = includes;
+    win.Object.defineProperty(Array.prototype, 'includes', {
+        enumerable: false,
```

Comment:
`enumerable` can be omitted, since it defaults to `false`. However, we should include `writable: true` and `configurable: true`.